topics/metabolomics/tutorials/mfassignr/tutorial.bib

```diff
@@ -0,0 +1,113 @@
+# you can use the 'online' format (see below)
+#
+# You can remove the examples below
+
+@article{Schum2020,
+  title = {MFAssignR: Molecular formula assignment software for ultrahigh resolution mass spectrometry analysis of environmental complex mixtures},
+  volume = {191},
+  ISSN = {0013-9351},
+  url = {http://dx.doi.org/10.1016/j.envres.2020.110114},
+  DOI = {10.1016/j.envres.2020.110114},
+  journal = {Environmental Research},
+  publisher = {Elsevier BV},
+  author = {Schum,  Simeon K. and Brown,  Laura E. and Mazzoleni,  Lynn R.},
+  year = {2020},
+  month = dec,
+  pages = {110114}
+}
+
+@article{Zheng2019,
+  title = {Resolution-enhanced Kendrick mass defect plots for the data processing of mass spectra from wood and coal hydrothermal extracts},
+  volume = {235},
+  ISSN = {0016-2361},
+  url = {http://dx.doi.org/10.1016/j.fuel.2018.08.085},
+  DOI = {10.1016/j.fuel.2018.08.085},
+  journal = {Fuel},
+  publisher = {Elsevier BV},
+  author = {Zheng,  Qingxin and Morimoto,  Masato and Sato,  Hiroaki and Fouquet,  Thierry},
+  year = {2019},
+  month = jan,
+  pages = {944–953}
+}
+
+@article{Kozhinov2013,
+  title = {Iterative Method for Mass Spectra Recalibration via Empirical Estimation of the Mass Calibration Function for Fourier Transform Mass Spectrometry-Based Petroleomics},
+  volume = {85},
+  ISSN = {1520-6882},
+  url = {http://dx.doi.org/10.1021/ac400972y},
+  DOI = {10.1021/ac400972y},
+  number = {13},
+  journal = {Analytical Chemistry},
+  publisher = {American Chemical Society (ACS)},
+  author = {Kozhinov,  Anton N. and Zhurov,  Konstantin O. and Tsybin,  Yury O.},
+  year = {2013},
+  month = jun,
+  pages = {6437–6445}
+}
+
+@article{Savory2011,
+  title = {Parts-Per-Billion Fourier Transform Ion Cyclotron Resonance Mass Measurement Accuracy with a “Walking” Calibration Equation},
+  volume = {83},
+  ISSN = {1520-6882},
+  url = {http://dx.doi.org/10.1021/ac102943z},
+  DOI = {10.1021/ac102943z},
+  number = {5},
+  journal = {Analytical Chemistry},
+  publisher = {American Chemical Society (ACS)},
+  author = {Savory,  Joshua J. and Kaiser,  Nathan K. and McKenna,  Amy M. and Xian,  Feng and Blakney,  Greg T. and Rodgers,  Ryan P. and Hendrickson,  Christopher L. and Marshall,  Alan G.},
+  year = {2011},
+  month = jan,
+  pages = {1732–1736}
+}
+
+@article{Green2015,
+  title = {Fast Graphically Inspired Algorithm for Assignment of Molecular Formulae in Ultrahigh Resolution Mass Spectrometry},
+  volume = {87},
+  ISSN = {1520-6882},
+  url = {http://dx.doi.org/10.1021/ac504166t},
+  DOI = {10.1021/ac504166t},
+  number = {10},
+  journal = {Analytical Chemistry},
+  publisher = {American Chemical Society (ACS)},
+  author = {Green,  Nelson W. and Perdue,  E. Michael},
+  year = {2015},
+  month = apr,
+  pages = {5086–5094}
+}
+
+@article{krevelen1950graphical,
+  title={Graphical-statistical method for the study of structure and reaction processes of coal.},
+  author={KREVELEN, VAN},
```

Comment:
please add a DOI or url field to this entry

Reply:
Its from the 50s, so unfortunately it doesn't have a DOI. Should I add some dummy DOI or so? Or will the linter be ok?

Reply:
I added a URL now and that seems to have been fine.